outputs/azureFunctions/withAzureFunctionsOutput.go

```diff
@@ -0,0 +1,12 @@
+package azureFunctions
+
+import "github.com/ls6-events/gengo"
+
+func WithAzureFunctionsOutput(directoryPath string) gengo.Option {
+	return func(s *gengo.Service) {
+		s.Outputs = append(s.Outputs, gengo.Output{
+			Mode:     gengo.OutputModeAzureFunctions,
+			Generate: generate(directoryPath),
```

Comment:
Please adapt to new code committed from PR [7!](https://www.github.com/LS6-Events/gengo/pull/7). Suggestion of also using the `IOConfiguration` style for caching the `directoryPath`?